adjustments.go

```diff
@@ -49,7 +49,7 @@ func (a Adjustment) MarshalXML(e *xml.Encoder, start xml.StartElement) error {
 		ProductCode       string   `xml:"product_code,omitempty"`
 		UnitAmountInCents int      `xml:"unit_amount_in_cents"`
 		Quantity          int      `xml:"quantity,omitempty"`
-		Currency          string   `xml:"currency"`
+		Currency          string   `xml:"currency,omitempty"`
```

Comment:
Can you add an in-line comment here that says something like `// Should only be omitted in purchase calls`? This field is still required to create an adjustment and I don't want it to look optional.